manual/histograms/index.md

```diff
@@ -197,84 +268,114 @@ Creating a new histogram without changing the original one:
+The following example shows how to normalize a histograms, such as it represents a probability density distribution
 
 _**Example**_
 
+The following histogram is given:
+
 {% highlight C++ %}
-   TH1F h1("h1","Histogram from a Gaussian",100,-3,3);
-   h1.FillRandom("gaus",10000);
-   h1.Draw();
+   TH1D *h = new TH1D("h","a trial histogram", 100, -1.5, 1.5);
+   for (Int_t i = 0; i < 10000; i++) h->Fill(gRandom->Gaus(0, 1));
+   h->Draw();
 {% endhighlight %}
 
-{% include figure_jsroot
-   file="histograms.root" object="h1" width="500px" height="350px"
-   caption="Histogram drawn with Draw()."
+{% include figure_image
+   img="histo-trial.png"
+   caption="A trial histogram for normalizing."
 %}
 
-#### Getting the bin width
-
-- Use the [GetBinWidth()](https://root.cern/doc/master/classTH1.html#ad69a5fa0002361fd77f37990a29d1aa3){:target="_blank"} method to get the bin width of a histogram.
+To use the normalization methods, you can clone first the histogram to keep the original one, call then `TH1::Scale` passing as scale parameter value the histogram integral. In addition, use the option `width` to divide also by the bin width in order to display the probability density in each bin.
+If you want to show just the frequency probability of each bin, you don't need to use the `width` option.
 
 {% highlight C++ %}
-   TH1F h1("h1","Histogram from a Gaussian",100,-3,3);
-   h1.FillRandom("gaus",10000);
-   h1.Draw();
-   h1.GetBinWidth(0)
+TH1*h1 = (TH1*)(h->Clone("h1"));
+h1->Scale(1./h1->Integral(), "width");
+h1->Draw();
+{% endhighlight %}
 
-   (double) 0.060000000
+
+After applying the normalization method, redraw the histogram with your preferred [drawing option](https://root.cern/doc/master/classTHistPainter.html#HP01):
+
+{% highlight C++ %}
+   myHist->Draw("HIST")
 {% endhighlight %}
 
-#### Drawing options
 
-> **Note**
->
-> The drawing options are not case sensitive.
 
-**Drawing options for all histogram classes**
+## Drawing a histogram
+
+- Use the [TH1::Draw()](https://root.cern/doc/master/classTH1.html#aa53a024a9e94d5ec91e3ef49e49563da){:target="_blank"} method to draw a histogram using the provided drawing option.
+    The drawing is delegated to the {% include ref class="THistPainter" %} class that specializes the drawing of the histogram. The {% include ref class="THistPainter" %} class is separated
+    from the histogram, so that the histogram class does not contain the graphics overhead.
+
+### Drawing options
 
 The "drawing option" is the unique parameter of the [TH1::Draw()](https://root.cern/doc/master/classTH1.html#aa53a024a9e94d5ec91e3ef49e49563da){:target="_blank"}
 method. It specifies how the histogram will be graphically rendered.
 For detailed information on the drawing options for all histogram classes, refer to
 [THistPainter](https://root.cern/doc/master/classTHistPainter.html#HP01a){:target="_blank"}.
```

Comment:
You can use: `{% include ref class="THistPainter" %}`

Reply:
Here I wanted to reference specific paragraph in the THistPainter class